### PR TITLE
Update ActScript.cs

### PR DESCRIPTION
**Pull request by fainagof**
Fix for issue in Scrip Action: In case of  FreeCommand execution fails with error "Script file not Selected . Kindly select suitable file".
Fix comment: Added verification for FreeCommand is empty or null.

### CODE REVIEW

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Ginger/GingerCore/Actions/ActScript.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4975222&bid=16431571)

File: Ginger/GingerCore/Actions/ActScript.cs [hunk 1]
```diff
@@ -141,10 +141,10 @@ protected void AddData(string outLine)
         }
         public override void Execute()
         {
-            if (ScriptName == null)
+            if ((ScriptName == null) && (GetInputParamCalculatedValue("Free Command") == null || GetInputParamCalculatedValue("Free Command") == string.Empty))
```
Comment (JinendraGhodke) [inline Ginger/GingerCore/Actions/ActScript.cs]: you can directly check string.IsNullorEmpty for input param free command instead of 2 conditions

Comment (JinendraGhodke) [reply Ginger/GingerCore/Actions/ActScript.cs:144]: Please confirm if you are handling this

**Comment by JinendraGhodke** [comment]
Closing as there is no update on this PR for a long time. If it is still relevant please reopen or start a new PR

**Pull request closed without merging**